packages/website/tests/index.spec.ts

```diff
@@ -1,7 +1,20 @@
 import AxeBuilder from '@axe-core/playwright';
-import { test } from '@playwright/test';
+import { expect, test } from '@playwright/test';
 
-test('Index', async ({ page }) => {
-  await page.goto('/');
-  await new AxeBuilder({ page }).analyze();
+test.describe('Website', () => {
+  test('Axe', async ({ page }) => {
+    await page.goto('/');
+    await new AxeBuilder({ page }).analyze();
+  });
+
+  test('should have no errors', async ({ page }) => {
+    const errorMessages = [] as string[];
```

Comment:
```suggestion
    const errorMessages: string[] = [];
```

We should really make a lint rule for this. I see people doing it a lot. #6374

Reply:
This would be a great lint rule, maybe I will give it a try after this PR.